### PR TITLE
Added a Classic Brick Breaker Game for Javascript Challenges

### PR DESCRIPTION
**Pull request by 5h0ov**
I added a classic Brick Breaker Game made using HTML, CSS and of course Javascript.

It is an interactive experience and of medium difficulty level to make it.

## **Working video 📹**
brick

Kindly review this PR under HackOctoberfest '23!

Thanks.

### CODE REVIEW

File: native/helpers/challenges.js [hunk 1]
```diff
@@ -100,6 +100,16 @@ export const challenges = [
...
     isNew: true,
   },
+  
+  {
+    title: 'Bricker Breaker Game',
+    link: 'brick_breaker',
+    difficulty: 'medium',
+    developer: '5h0ov',
+    tags: [],
+    youtube: 'https://www.youtube.com/channel/UCYWgCjYT8_05-rLq0Xmbm-Q',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:110]: only video link of the solution to the challenge is allowed. Please keep it empty if there is no link

**Review by sadanandpai — APPROVED** [review]

File: native/helpers/challenges.js (hunk 1 above)
Comment (5h0ov) [reply native/helpers/challenges.js:110]: oh i see oops sorry hold on

Comment (5h0ov) [reply native/helpers/challenges.js]: done sir @sadanandpai

Comment (sadanandpai) [reply native/helpers/challenges.js]: @5h0ov The experience is completely broken for mobile users

**Comment by sadanandpai** [comment]
@5h0ov Please work on the responsive part as it broken completely

**Comment by 5h0ov** [comment]
> @5h0ov Please work on the responsive part as it broken completely

I have managed to make the play area be within the device resolution:
image


Will try my best to make the paddle move on touch in mobile screens.

**Pull request merged**